docs/src/getting-started/cash-app-pay.md

```diff
@@ -0,0 +1,211 @@
+| `.integrationError` | A fixable bug in your integration |
+| `.apiError` | A degradation of Cash App Pay server APIs. Your app should temporarily hide Cash App Pay functionality |
+| `.unexpectedError` | A bug outside the normal flow. Report this bug (and what caused it) to Cash App Developer Support |
+
+{: .info }
+> You must update your UI in response to these state changes
+
+## Step 3: Implement URL handling
+
+To use PayKit iOS, Cash App must be able to call a URL that will redirect back to your app. The simplest way to accomplish this is via [Custom URL Schemes][custom-url-schemes], but if your app supports [Universal Links][universal-links] you may use those URLs as well.
+
+Choose a unique scheme for your application and register it in Xcode from the Info tab of your application’s Target.
+
+You’ll pass a URL that uses this scheme (or a Universal Link your app handles) into the `createCustomerRequest()` method that starts the authorization process.
+
+When your app is called back by Cash App, simply post the `PayKit.RedirectNotification` from your AppDelegate or SceneDelegate, and the SDK will handle the rest:
+
+``` swift
+import UIKit
+import PayKit
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+  func scene(_ scene: UIScene, openURLContexts URLContexts: Set<UIOpenURLContext>) {
+    if let url = URLContexts.first?.url {
+      NotificationCenter.default.post(
+        name: PayKit.RedirectNotification,
+        object: nil,
+        userInfo: [UIApplication.LaunchOptionsKey.url : url]
+      )
+    }
+  }
+}
+```
+
+## Step 4: Instantiate PayKit iOS
+
+When you’re ready to authorize a payment using Cash App Pay,
+
+1. Instantiate the SDK with the Afterpay PayKit Client ID via `Afterpay.cashAppClientId`
+2. The SDK defaults to point to the production endpoint; for development, set the endpoint to `.sandbox`
+3. Add your observer to the PayKit SDK
+
+{: .note }
+> Ensure that the Afterpay SDK is configured per the [instructions][afterpay-configuration] before attempting to access `Afterpay.cashAppClientId`
+
+For example, from your checkout view controller that implements the `PayKitObserver` protocol, you might instantiate the SDK to be:
+
+``` swift
+private let sandboxClientID = Afterpay.cashAppClientId
+private lazy var sdk: PayKit = {
+    let sdk = PayKit(clientID: sandboxClientID, endpoint: .sandbox)
+    sdk.addObserver(self)
+    return sdk
+}()
+```
+## Step 5: Create a Customer Request
+
+You can create a customer request as soon as you know the amount you’d like to charge or if you'd like to create an on-file payment request. You’re encouraged to create this request as soon as your checkout view controller loads, so that your Customer can authorize the request without delay.
+
+### Step 5A: Sign the Order Token
+
+After retrieving the token from your server-to-server call, the order needs to be signed, so as to retrieve the JWT and associated data. This might look something like the below:
```

Comment:
```suggestion
After retrieving the token from your server-to-server call, the order must be signed, so that you can retrieve the JWT and associated data. This might look like the following example:
```